apps/intellij/src/main/kotlin/dev/nx/console/nx_toolwindow/NxToolWindowPanel.kt

```diff
@@ -1,93 +1,231 @@
+                                    ) {
+                                        if (it.description == "open-setting") {
+                                            ShowSettingsUtil.getInstance()
+                                                .showSettingsDialog(
+                                                    project,
+                                                    NxConsoleSettingsConfigurable::class.java
+                                                )
+                                        } else {
+                                            Desktop.getDesktop().browse(URI.create(it.description))
+                                        }
+                                    }
+                                    .align(Align.CENTER)
+                            }
+                        }
+                    )
+                }
+            )
+        }
+    }
+
+    private fun createErrorComponent(errorCount: Int): JComponent {
         return panel {
             indent {
                 row {
                     text(
-                        "<h3>We couldn't find any projects in this workspace.</h3> Make sure that the proper dependencies are installed locally and refresh the workspace."
+                        "<h3> Nx caught ${if(errorCount == 1) "an error" else "$errorCount errors"} while computing the project graph.</h3>"
                     )
                 }
                 row {
-                    button("Refresh Workspace", NxRefreshWorkspaceAction(), NX_TOOLBAR_PLACE)
+                    button("View Errors") { ProblemsView.getToolWindow(project)?.show() }
                         .align(Align.CENTER)
                 }
                 row {
                     text(
-                        "If you're just getting started with Nx, you can <a href='https://nx.dev/plugin-features/use-code-generators'>use generators</a> to quickly scaffold new projects or <a href='https://nx.dev/reference/project-configuration'>add them manually</a>." +
-                            "<br/> If your Nx workspace is not at the root of the opened project, make sure to set the <a href='open-setting'>workspace path setting</a>."
+                        "If the problems persist, you can try running <code>nx reset</code> to clear all caches and then <a href='refresh'>refresh the workspace</a><br /> For more information, look for errors in <a href='open-idea-log'>idea.log</a> and refer to the <a href='https://nx.dev/troubleshooting/troubleshoot-nx-install-issues?utm_source=nxconsole'>Nx Troubleshooting Guide </a> and the <a href='https://nx.dev/recipes/nx-console/console-troubleshooting?utm_source=nxconsole'>Nx Console Troubleshooting Guide</a>."
```

Comment:
I don't think we need to say `to clear all caches`, just `try running nx reset and then refresh...`.